spec/unit/basket_spec.rb

```diff
@@ -4,10 +4,12 @@
 RSpec.describe Basket do
 
   context "#initialize" do
+
     it "creates correctly" do
       expect(Basket.new.products).to_not eql(nil)
```

Comment:
it's better to check what `Basket.new.products` is (eql what) not just it's not `nil`